spec/features/edit_scanned_resource_spec.rb

```diff
@@ -45,27 +45,6 @@
       expect(page).to have_text("Error retrieving metadata")
     end
 
-    scenario "User can add a new file" do
```

Comment:
Would it be possible to rework this test to work with the File Manager?  It would be great to keep a test that actually uploaded a file.